version.rb

```diff
@@ -5,6 +5,6 @@ module VERSION #:nodoc:
     TINY  = 22
     PRE   = "2"
 
-    STRING = [MAJOR, MINOR, TINY, PRE].compact.join('.')
```

Comment:
No idea if this is a good idea.  I am going to guess it is not.   I just assume that there is code looking at the Rails Version that this will confuse.

I assume this is to check if this is the invoca version.  Just define a constant just above it that we can check exists.